src/routes/webhooks/paddle.ts

```diff
@@ -0,0 +1,112 @@
+import { FastifyInstance } from 'fastify';
+import {
+  Environment,
+  EventName,
+  Paddle,
+  SubscriptionCanceledEvent,
+  SubscriptionCreatedEvent,
+} from '@paddle/paddle-node-sdk';
+import createOrGetConnection from '../../db';
+import { updateSubscriptionFlags } from '../../common';
+import { User } from '../../entity';
+import { logger } from '../../logger';
+
+const paddleInstance = new Paddle(process.env.PADDLE_API_KEY, {
+  environment: Environment.sandbox,
```

Comment:
Just making sure we want sandbox environment

Reply:
Nope, I'll make this env based as well :)